### PR TITLE
add missing dependencies README.md

### PR DESCRIPTION
**Pull request by Gadgeteering**
Add missing dependencies to README.md and correction of the yarn install instruction

### CODE REVIEW

**Comment by jyjblrd** [comment]
Thanks for this, I left some comments. I'll merge it once you make the changes

**Comment by Gadgeteering** [comment]
Not sure why but I can't see your comments

**Comment by jyjblrd** [comment]
my bad! forgot to submit the review

File: README.md [hunk 1]
```diff
@@ -9,8 +9,17 @@ This project requires the sfm (structure from motion) OpenCV module, which requi
...
 `cv.sfm` is only used 3 times in the codebase for the following functions: `fundamentalFromProjections`, `essentialFromFundamental`, `motionFromEssential`. So really, those functions should just be reimplemented in Python so the sfm module isn't needed. [Issue](https://github.com/jyjblrd/Mocap-Drones/issues/4).
 
+On OSX install yarn `brew install yarn` 
+
+pip install flask_socketio
+
+pip install ruckig
+
+pip install flask-cors
+
```
[the remark this answers is not in the document]
Comment (Gadgeteering) [reply README.md:13]: I have remove all pip commands might be worth creating a requirement.txt file in the future and use pip install -r requirements.txt

File: README.md [hunk 2]
```diff
@@ -9,8 +9,17 @@ This project requires the sfm (structure from motion) OpenCV module, which requi
 
 `cv.sfm` is only used 3 times in the codebase for the following functions: `fundamentalFromProjections`, `essentialFromFundamental`, `motionFromEssential`. So really, those functions should just be reimplemented in Python so the sfm module isn't needed. [Issue](https://github.com/jyjblrd/Mocap-Drones/issues/4).
 
+On OSX install yarn `brew install yarn`
```
[the remark this answers is not in the document]
Comment (Gadgeteering) [reply README.md]: I have changed this to a generic term

**Review by jyjblrd — APPROVED** [review]

**Pull request merged**